flagscale/runner/runner_serve.py

```diff
@@ -162,10 +160,17 @@ def _prepare(self):
         _update_config_serve(self.config)
         self.user_args = _get_args_vllm(self.config)
         self.user_envs = self.config.experiment.get("envs", {})
+        entrypoint = self.config.experiment.task.get("entrypoint")
         if self.command_line_mode:
             self.user_script = "flagscale/serve/run_vllm.py"
+        elif isinstance(entrypoint, str) and entrypoint.endswith(".py"):
+            self.user_script = entrypoint
+        elif entrypoint is None:
```

Comment:
Should we require users to provide the entrypoint?

Reply:
This is an option. Models pipeline can be built via user's entrypoint file or yaml config.